bitwarden_license/bit-web/src/app/secrets-manager/settings/porting/sm-export.component.html

```diff
@@ -10,7 +10,7 @@
   <bit-form-field class="tw-max-w-sm">
     <bit-label>{{ "fileFormat" | i18n }}</bit-label>
     <select bitInput formControlName="format">
-      <option *ngFor="let format of exportFormats" [ngValue]="format">{{ format }}</option>
+      <option *ngFor="let format of exportFormats">{{ format.name }}</option>
```

Comment:
We could use the index here instead which simplifies the logic.

```suggestion
      <option *ngFor="let format of exportFormats; let i = index" [value]="index">{{ format.name }}</option>
```

Reply:
Updated